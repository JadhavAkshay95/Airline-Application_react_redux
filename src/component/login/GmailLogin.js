import React from 'react';
import GoogleLogin from 'react-google-login';
import PropTypes from 'prop-types';

const GmailLogin = ({ handleLogin }) => {
  return (
    <GoogleLogin
      clientId="37487767547-1g4akkokm58ig8k1et84qm50tpoo891b.apps.googleusercontent.com"
      buttonText="LOGIN WITH GOOGLE"
      onSuccess={handleLogin}
    />
  );
};

export default GmailLogin;
GmailLogin.propTypes = { handleLogin: PropTypes.func };
