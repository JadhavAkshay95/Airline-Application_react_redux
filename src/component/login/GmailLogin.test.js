import React from 'react'
import { shallow } from 'enzyme'
import GmailLogin from './GmailLogin'

const renderGmailLogin = args => {
  const defaultProps = { handleLogin: () => {} }
  const props = { ...defaultProps, ...args }
  return shallow(<GmailLogin {...props} />)
}

describe('Render google login component', () => {
  it('Render flight list', () => {
    const wrapper = renderGmailLogin()
    expect(wrapper).toMatchSnapshot()
  })
})
