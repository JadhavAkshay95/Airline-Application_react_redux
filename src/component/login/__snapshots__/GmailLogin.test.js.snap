// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render google login component Render flight list 1`] = `ShallowWrapper {}`;
